### ISSUE TITLE
Not compatible to SQANTI3 v5.1

### ISSUE THREAD

**Issue by kehming** [nch-igm/PBFLIP#2]
Hi,

Since SQANTI3 has been updated to version 5.1, here the PBFLIP is not compatible with it.
Would you consider to update PBFLIP soon to run with SQANTI3 v5.1?
Meanwhile, the crucial FusionHubDB database file (https://fusionhub.persistent.co.in/out/global/Result_0_1000.html) is unavailable !!!
What happened to the FusionHubDB ??

KM

**Comment by wijerasa** [comment]
> Hi,
> 
> Since SQANTI3 has been updated to version 5.1, here the PBFLIP is not compatible with it. Would you consider to update PBFLIP soon to run with SQANTI3 v5.1?

Yes, we are considering to update the pb_flip with new version of SQANTI3 but I don't have a definite time frame for the upgrade at this moment. 
>
 >Meanwhile, the crucial FusionHubDB database file 
(https://fusionhub.persistent.co.in/out/global/Result_0_1000.html) is unavailable !!! What happened to the FusionHubDB ??

The database has been migrated to a new web page and I have updated the `README.md` file to reflect the new web page. Let me know if that doesn't work for you.

Thank you!
